readme.md

```diff
@@ -4,40 +4,64 @@
 
 Debug HTTP printer for pino
 
-## Supports
+```sh
+[1574071926285] GET http://localhost:20000/api/activity/component 200\n
+```
+
+## Usage as Pino Transport
```

Comment:
Let's add a note that it can only be at the end of a pipeline chain. Ask @mcollina for details.

Reply:
At the moment:

- every pino 7 transport can be used at the end of the pipeline
- only pino-syslog has the `enablingPipeline` flag that is documented

so, I see the pattern:

- the user can be pipelining only if the plugin supports the `enablingPipeline` flag
- if the module does not support it, it is not documented

For these reasons I would avoid this info since it can cause confusion.
The input of this plugin cannot be a string in the `syslog` format

Reply:
I agree with @Eomm point of view. I don't think we should add this here.

Are you asking to add it to all other transports too?